src/oic/oic/provider.py

```diff
@@ -2197,7 +2197,13 @@ def end_session_endpoint(
             try:
                 _ruri = self.cdb[client_id]["post_logout_redirect_uris"]
             except KeyError:
-                redirect_uri = self.post_logout_page
+                if self.post_logout_page is None:
+                    logger.info("No post logout page configured for %s", client_id)
```

Comment:
IMHO this should rather be a warning than info.

Reply:
OK